web/modules/custom/joinup_event/config/install/core.entity_form_display.node.event.default.yml

```diff
@@ -32,11 +32,28 @@ dependencies:
     - allowed_formats
     - comment
     - datetime_range
+    - field_group
     - file
     - image
     - joinup_core
     - link
     - text
+third_party_settings:
+  field_group:
+    group_event_location:
+      children:
+        - field_location
+        - field_event_online_location
+      parent_name: ''
+      weight: 0
```

Comment:
The new field group is placed on top of form:
<img width="974" alt="loc" src="https://user-images.githubusercontent.com/473868/55192741-dcf8dd80-51ad-11e9-9259-1a7195811ef5.png">

It should be placed in the same location as the physical location was before this change.

Reply:
"Move the location's fields in a section with header: Event Location."
Maybe it was confusing me a bit, so what I actually should do here?

Reply:
It's OK, just the section position is wrong. It's now on top of the form

Reply:
Okay moved to the original place!